I/ICU/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "ICU"
+version = v"64.2"
+
+# Collection of sources required to build ICU
+sources = [
+    "http://download.icu-project.org/files/icu4c/$(version.major).$(version.minor)/icu4c-$(version.major)_$(version.minor)-src.tgz" =>
+    "627d5d8478e6d96fc8c90fed4851239079a561a6a8b9e48b0892f24e82d31d6c",
+
+    "https://github.com/giordano/Yggdrasil/releases/download/icu/ICU_native_build.tar.xz" =>
```

Comment:
For context, this is a native for `x86_64-linux-gnu` that I did locally, and put in on GitHub to make it easier to reuse.  I have simply archived the `icu/source` (renamed `native` here, if I remember correctly) directory after the build.  Using the tarball automatically produced after installation doesn't work, because some files in the `config` directory are necessary, and this directory is not installed in `${prefix}`.

Reply:
Ok, no more need for this